webpack/assets/javascripts/react_app/redux/index.js

```diff
@@ -25,8 +26,6 @@ const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 export const generateStore = () =>
   createStore(reducers, composeEnhancers(applyMiddleware(...middleware)));
 
-window.tfm_redux_store = window.tfm_redux_store || generateStore();
-
-const store = window.tfm_redux_store;
+const store = forceSingleton('redux_store', () => generateStore());
```

Comment:
Would this work the same?
```suggestion
const store = forceSingleton('redux_store', generateStore);
```

Reply:
Yes, replaced